frontend/vite.config.ts

```diff
@@ -1,9 +1,48 @@
+import { readFileSync } from 'node:fs'
 import { resolve } from 'node:path'
 import { execSync } from 'child_process'
 import { defineConfig, loadEnv } from 'vite'
 import solidPlugin from 'vite-plugin-solid'
 import unocssPlugin from 'unocss/vite'
 
+function eruda(debug = undefined) {
+  const erudaSrc = readFileSync("./node_modules/eruda/eruda.js", "utf-8");
+  return {
+    name: "vite-plugin-eruda",
+    transformIndexHtml(html) {
+      const tags = [
+        {
+          tag: "script",
+          children: erudaSrc,
+          injectTo: "head",
+        },
+        {
+          tag: "script",
+          children: "eruda.init();",
+          injectTo: "head",
+        },
+      ];
+      if (debug === true) {
+        return {
+          html,
+          tags,
+        };
+      } else if (debug === false) {
```

Comment:
why not a normal else?

Reply:
because if debug == undefined it would go in that branch and return  a value instead of checking NODE_ENV in the next `if` statement